packages/mobile/src/verify/saga.ts

```diff
@@ -795,6 +807,7 @@ export function* receiveAttestationCodeSaga(action: ReturnType<typeof receiveAtt
       `Attestation code (${message}) is valid, starting processing (issuer: ${issuer})`
     )
 
+    yield put(setAttestationInputStatus({ index, status: CodeInputStatus.Processing }))
```

Comment:
I lied in the previous comment, I also added this which is necessary so that users can input more codes while the first one completes.

Reply:
When codes are input there are two steps: 
- The first one is this function which does some basic and quick checks. 
- The next one is the complete saga, which calls komenci to send the codes.

When a user puts in a code, we want them to wait until we do the first few checks before going to the next input. Otherwise, if these checks fail the user will already have the next input focused but the previous one will also be available because it failed. This is captured by the `CodeInputStatus`, `Received` means doing some quick checks and `Processing` means sending the codes to Komenci.

There might be a better way of doing this, maybe we can just make this quick checks in the UI so that the `Received` status is no longer necessary (we'd have to check automated code-pasting from android too) but the UI still depends on these index-based mappings unfortunately :(

Reply:
I see. Is there still value in restricting inputs though? Why do we even focus code inputs one by one instead of just making them all available at the same time? Maybe this was to make sure that codes were inputting sequentially but now that we have mutex locks it may just be adding unnecessary complexity

Reply:
Oh, I had missed this comment. I'm not sure, maybe a UX decision to give users clarity on what they have to do? If they have the 3 options they may not know where to put the code or something. Definitely worth thinking about this